src/lib/style/components/_all.scss

```diff
@@ -5,19 +5,19 @@
   './total-query-results', './facet_nav/all', './query-wrapper-plot-nav',
   './query-wrapper-text-input', './element-with-tooltip',
   './_column-selection.scss', './programmatic-options',
-  './query_filter/enum-facet-filter', './error', './goals', './feed-cards',
-  './upset-plot', './resources', 'cards-rotate', './expandable_content',
-  './bar-loader', './facet-plots-card', './featured-data-plots',
-  './featured-data-tabs', './programs', './user-card-list-groups',
-  './evaluation_queue/all', './personal-access-tokens', './copy-to-clipboard',
-  './account-level-badge', './terms-and-conditions', './page-progress',
-  './project-view-card', './carousel', './synapse-homepage', './login',
-  './entity_finder/all', './entity-badge', './download-list-v2',
-  './experimental-mode', './user-search-box', './entity-metadata',
-  './download-cart-page', './global-alert', './annotations', 'plotlywrapper',
-  './synapse-nav-drawer', './favorites-page', './skeletons', './toasts',
-  './user-profile-links', './blocking-loader', './transition', './popover',
-  './featured-tool-card', './row_renderers/observation-card',
-  './subsection-row-renderer', './dataset-editor', './fluid-modal',
-  './markdown-collapse', './discussion-search', './submission-page',
-  './governance-dashboard';
+  './query_filter/enum-facet-filter', './query_filter/facet-filter-header',
```

Comment:
`./query_filter/facet-filter-header` was imported inline, just making sure to import it via the full CSS build